### PR TITLE
6 new and 1 modified 2015+ GHSA-sync'ed files

### PR DESCRIPTION
**Pull request by jasnow**
6 new and 1 modified 2015+ GHSA-sync'ed files

* Ran the unchanged "lib/sync_github_advisories" to get the most recent files in the query.
* Did minor reformatting (line wrapping) but no new content.
* Ran "yamllint" and "rake" (both clean/green).

### CODE REVIEW

File: gems/audited/GHSA-hjp3-5g2q-7jww.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+patched_versions:
+  - '>= 5.3.3'
+related:
+  url:
+    - https://github.com/collectiveidea/audited/security/advisories/GHSA-hjp3-5g2q-7jww
+    - https://github.com/collectiveidea/audited/issues/601
+    - https://github.com/collectiveidea/audited/pull/669
+    - https://github.com/collectiveidea/audited/pull/671
+    - https://github.com/advisories/GHSA-hjp3-5g2q-7jww
+notes: 'vulnerableVersionRange: ">= 4.0.0, < 5.3.3";   firstPatchedVersion: 5.3.3'
```
Comment (postmodern) [inline gems/audited/GHSA-hjp3-5g2q-7jww.yml:37]: `notes:` should be human readable text, also this information is already captured in `patched_versions` and `unaffected_versions`. I think it's safe to omit `notes:` from these advisories.

File: gems/govuk_tech_docs/GHSA-x2xw-hw8g-6773.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+gem: govuk_tech_docs
+ghsa: x2xw-hw8g-6773
+url: https://github.com/alphagov/tech-docs-gem/security/advisories/GHSA-x2xw-hw8g-6773
+title: govuk_tech_docs vulnerable to unescaped HTML on search results page
+date: 2023-04-11
+description: |-
+  'Impact
```
Comment (postmodern) [inline gems/govuk_tech_docs/GHSA-x2xw-hw8g-6773.yml:8]: Single quotes around block text is unnecessary.

File: gems/audited/GHSA-hjp3-5g2q-7jww.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+gem: audited
+ghsa: hjp3-5g2q-7jww
+url: https://github.com/collectiveidea/audited/security/advisories/GHSA-hjp3-5g2q-7jww
+title: Race Condition leading to logging errors
+date: 2023-05-01
+description: |
+  'In certain setups with threaded web servers, Audited's use of
```
Comment (postmodern) [inline gems/audited/GHSA-hjp3-5g2q-7jww.yml:8]: Should be safe to remove surrounding single-quotes from `|` block-text.

**Review by postmodern — CHANGES_REQUESTED** [review]
Minor stylistic changes needed.

File: gems/audited/GHSA-hjp3-5g2q-7jww.yml (hunk 1 above)
Comment (jasnow) [reply gems/audited/GHSA-hjp3-5g2q-7jww.yml:37]: README says "internal notes" - I'm showing the before text before I manually converted it to your format.

File: gems/audited/GHSA-hjp3-5g2q-7jww.yml (hunk 3 above)
Comment (jasnow) [reply gems/audited/GHSA-hjp3-5g2q-7jww.yml]: fixed

File: gems/govuk_tech_docs/GHSA-x2xw-hw8g-6773.yml (hunk 2 above)
Comment (jasnow) [reply gems/govuk_tech_docs/GHSA-x2xw-hw8g-6773.yml]: fixed

File: gems/audited/GHSA-hjp3-5g2q-7jww.yml (hunk 1 above)
Comment (jasnow) [reply gems/audited/GHSA-hjp3-5g2q-7jww.yml]: fixed

**Comment by jasnow** [comment]
Ready for re-review

**Review by postmodern — APPROVED** [review]
LGTM

**Comment by jasnow** [comment]
Thanks

**Pull request merged**